### PR TITLE
docs: update README with new color

### PR DESCRIPTION
**Pull request by mrhappyma**
This was randomly irritating me so I figured I would fix it myself :)

One change I wasn't sure about, the Discord logo was previously from `http://fig.io/icons/discord-logo-square.png`, I wasn't quite sure where to put a new one so it is within the `.github` folder. Was there a specific reason it was pulled from the site?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ If you aren't able to contribute, please feel free to open an [issue](https://gi
   Join our community
   <br/>
   <a href="https://fig.io/community">
-    <img alt="Discord logo" src="http://fig.io/icons/discord-logo-square.png" width="80px" height="80px" />
+    <img alt="Discord logo" src="./.github/discord-square.png" width="80px" height="80px" />
```
Comment (fedeci) [inline README.md:138]: I suppose we should update the icon on the website

Comment (fedeci) [reply README.md:138]: Opened a PR in the website repo /cc @SeparateRecords

Comment (mrhappyma) [reply README.md]: Updated PR to just change badge color

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**